manual/manual/tutorials/coreexamples.etex

```diff
@@ -580,6 +580,80 @@ let fixpoint f x =
 the function "f" cannot raise a "Done" exception, which removes an
 entire class of misbehaving functions.
 
+\section{Lazy expressions}
+
+OCaml allows us to defer some computation, usually expensive one, until later when 
+we need the result of that computation. 
+
+We use "lazy (expr)" to delay the evaluation of some expression "expr". For 
+example, we can defer the computation of "1+1" until we need the result of that
+expression, i.e. "2". Let us see how we initialize a lazy expression. 
+
+\begin{caml_example}{toplevel}
+  let lazy_two = lazy ( print_endline "lazy_two evaluation"; 1 + 1 );;
+\end{caml_example}
+
+We added "print_endline \"lazy_two evaluation\"" to see when the lazy
+ expression is being evaluated.
+
+There are several things to note about the result printed by the toplevel above. 
+First, "lazy_two" has type "int lazy_t". However, "'a lazy_t" is a type 
+constructor for keyword "lazy" used by the compiler. 
+One should use type "'a Lazy.t" to work with lazy expressions. Second, the value 
+of "lazy_two" is "<lazy>", which means the expression is not evaluated yet, and 
+the result is unknown.
+
+When we finally need the result of a lazy expression, we can call "Lazy.force"  
+on that expression to force its evaluation. "force" is a function from a 
```

Comment:
If possible I prefer to avoid starting sentence with a variable names like `force`. For instance, the above could be rewritten as 

> The function "force" comes from